packages/web/src/client/components/Grids/GridValues.stories.tsx

```diff
@@ -1,13 +1,70 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react'
+
 import { ValuesGrid } from './GridValues'
-import { Story, Meta } from '@storybook/react'
 
-const Template: Story = (args) => <ValuesGrid {...args} />
+export default {
+  title: 'Values Grid',
+  component: ValuesGrid,
+} as ComponentMeta<typeof ValuesGrid>
 
-export const Primary = Template.bind({})
+export const NoValues: ComponentStory<typeof ValuesGrid> = () => (
+  <ValuesGrid qualities={fixtures.noValues} />
+)
 
-Primary.args = {}
+export const WithValues: ComponentStory<typeof ValuesGrid> = () => (
+  <ValuesGrid qualities={fixtures.withValues} />
+)
 
-export default {
-  title: 'Components/Values Grid',
-  component: ValuesGrid,
-} as Meta
+const fixtures = {
+  noValues: undefined,
+  withValues: [
+    {
```

Comment:
I would add more values so we can see this a grid. Equally a single value is a good idea for a variation.